library/src/scripts/forums/forumStyleVars.ts

```diff
@@ -29,6 +30,29 @@ export const forumVariables = useThemeCache((forcedVars?: IThemeVariables) => {
                 left: "initial",
             },
         },
+        colors: {
+            backgroundColor: "transparent",
+            backgroundColorRead: "transparent",
+        },
+        text: {
+            title: {
+                colors: {
+                    noState: "",
+                    focus: "",
+                    hover: "",
+                    active: "",
+                },
+            },
+            meta: {
+                colors: {
+                    noState: colorOut(globalVars.mainColors.fg),
+                    focus: colorOut(globalVars.links.colors.focus),
+                    keyboardFocus: colorOut(globalVars.links.colors.keyboardFocus),
+                    hover: colorOut(globalVars.links.colors.hover),
+                    active: colorOut(globalVars.links.colors.active),
```

Comment:
These are not styles so I would recommend against using `colorOut` here. You can just use the color directly.

`colorOut()` is to be used only when creating the final CSS class.